pyaedt/modules/Boundary.py

```diff
@@ -803,12 +1001,42 @@ def identify_extent_poly(self):
             return False
         return outlines[0].name
 
+    @property
+    def board_cutout_material(self):
+        """Material applied to cutouts regions."""
+        return self.props["NativeComponentDefinitionProvider"].get("BoardCutoutMaterial", "air ")
+
+    @property
+    def via_holes_material(self):
+        """Material applied to via holes regions."""
+        return self.props["NativeComponentDefinitionProvider"].get("ViaHoleMaterial", "copper")
+
+    @board_cutout_material.setter
+    def board_cutout_material(self, value):
+        """Set material to apply to cutouts regions.
+
+        Parameters
+        ----------
+        value : str
+            Material to apply to cutouts regions.
+        """
+        self.props["NativeComponentDefinitionProvider"]["BoardCutoutMaterial"] = value
+
+    @via_holes_material.setter
+    def via_holes_material(self, value):
+        """Set material to apply to via holes regions.
```

Comment:
Same questions here about the "s" in holds. In the parameter description, regions isn't mentioned. Is this an oversight?